### PR TITLE
Pass Publication.info on Java level

### PR DESCRIPTION
**Pull request by oleg-smith**

### CODE REVIEW

**Comment by oleg-smith** [comment]
@FZambia could you please review

File: centrifuge/src/main/java/io/github/centrifugal/centrifuge/Publication.java [hunk 1]
```diff
@@ -20,4 +20,14 @@ void setOffset(long offset) {
...
 
     private long offset;
+
+    private ClientInfo info;
+
+    public ClientInfo getInfo() {
+        return info;
+    }
+
+    public void setInfo(ClientInfo info) {
```
Comment (FZambia) [inline centrifuge/src/main/java/io/github/centrifugal/centrifuge/Publication.java:30]: should not be public

Comment (oleg-smith) [reply centrifuge/src/main/java/io/github/centrifugal/centrifuge/Publication.java:30]: fixed

**Comment by FZambia** [comment]
Thank you!

**Comment by oleg-smith** [comment]
@FZambia can it be released now or there is a certain cadence?

**Comment by FZambia** [comment]
Yep - https://github.com/centrifugal/centrifuge-java/releases/tag/0.4.1

**Pull request merged**